tests/L0/run_transformer/test_pipeline_parallel_fwd_bwd.py

```diff
@@ -112,6 +139,14 @@ def _forward_backward_test_impl(
                 deallocate_pipeline_output=deallocate_pipeline_outputs,
             )
 
+            hidden_size = PipelineParallelForwardBackwardTest.HIDDEN_SIZE
+            microbatch_size = PipelineParallelForwardBackwardTest.MICRO_BATCH_SIZE
+            target_loss = get_target_loss(hidden_size, microbatch_size, pipeline_model_parallel_world_size, self.world_size)
+
+            for loss_item in loss:
+                x = loss_item['avg']
+                torch.testing.assert_close(x, target_loss*torch.ones_like(x))
```

Comment:
and as dtype can be torch.bfloat16/torch.float16, I want to guard this nice numerical check
```suggestion
            if dtype == torch.float32:
                for loss_item in loss:
                    x = loss_item['avg']
                    torch.testing.assert_close(x, target_loss*torch.ones_like(x))
```